Release/include/cpprest/details/http_client_impl.h

```diff
@@ -57,6 +57,11 @@ static utility::string_t flatten_http_headers(const http_headers &headers)
     utility::string_t flattened_headers;
     for(auto iter = headers.begin(); iter != headers.end(); ++iter)
     {
+        utility::string_t header_name = iter->first;
+        http_headers::_case_insensitive_cmp cmp;
+        if (!cmp(header_name, header_names::host) && !cmp(header_names::host, header_name)) {
```

Comment:
Also, out windows http client implementation is in http_client_winhttp.cpp. We should make sure that we dont break anything there when modifying these common files.